netjson-monitoring.lua

```diff
@@ -25,6 +25,40 @@ function split(str, pat)
    return t
 end
 
+-- parse /proc/net/arp
+function parse_arp()
+   arp_info = {}
+   for line in io.lines('/proc/net/arp') do
+      if line:sub(1, 10) ~= 'IP address' then
+         ip, hw, flags, mac, mask, dev = line:match("(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)%s+(%S+)")
+         table.insert(arp_info, {
+            ip_address = ip,
+            mac_address = mac,
+            interface = dev,
+            type = hw,
+            flags = flags,
+            mask = mask
+         })
+      end
+   end
+   return arp_info
+end
+
+function get_ip_neigh()
+   arp_info = {}
+   output = io.popen('ip -json neigh'):read()
```

Comment:
we need to manage the case in which the command is not available, I'd return `false` if the program is not available. This should help: https://www.lua.org/pil/8.4.html

Reply:
I've decided to check if `sbin/ip` exists because io.popen('ip') does not raise any error

Reply:
Update: I don't even check if the file exists, checking if the returned value is nil seems to be enough

Reply:
Ok, that's fine, thanks!